src/Map/src/Live/ComponentWithMapTrait.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Live;
+
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\Map\Map;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+use Symfony\UX\TwigComponent\Attribute\PostMount;
```

Comment:
Can we do without at first or is it required ? (genuine question)

Reply:
I think it's nice to keep it